core/pkg/opencost/allocation.go

```diff
@@ -97,7 +97,9 @@ type Allocation struct {
 	// UnmountedPVCost is used to track how much of the cost in PVs is for an
 	// unmounted PV. It is not additive of PVCost() and need not be sent in API
 	// responses.
-	UnmountedPVCost float64 `json:"-"` //@bingen:field[ignore]
+	UnmountedPVCost   float64 `json:"-"`                 //@bingen:field[ignore]
+	GPURequestAverage float64 `json:"gpuRequestAverage"` //@bingen:field[version=22]
+	GPUUsageAverage   float64 `json:"gpuUsageAverage"`   //@bingen:field[version=22]
```

Comment:
I added these new fields to the bottom of the struct in accordance with https://github.com/opencost/opencost/blob/develop/core/pkg/opencost/bingen.go.

Reply:
I've noticed that the codecs performs "field version checks". Which leads me to believe that we don't _have_ to add new fields to the end of the struct. If possible, I'd like to group these new fields alongside the other GPU fields.